bin/disclosure

```diff
@@ -69,30 +93,39 @@ puts "Found #{input_emails.size} emails"
 
 parsed_emails = []
 
-input_emails.each do |email_path|
-  # TODO if we fail parsing a single email, write out the progress so far
-  # to the output json file and allow picking back up where we left off on
-  # next run
-  message = Message.new(email_path)
-
-  # Write attachments out
-  message.attachments.each do |attachment|
-    # TODO convert to PDF using soffice headless?
-    #
-    # TODO handle duplicate clashing filenames, update attachment filename in output
-    # JSON if needed
-    attachment.write_to(File.join(options[:attachment_dir], attachment.filename))
-  end
+input_emails[options[:offset] .. -1].each_with_index do |email_path, i|
```

Comment:
Is `input_emails` guaranteed to be sorted in same order between each run, or should we call `sort` on the input array each time just to make sure?